changelog

```diff
@@ -1,3 +1,95 @@
+Version 0.9.11:
+  * Big changes:
+    * Added a new area, the Graveyard, a region of the Ember Waste inhabited by a new hostile alien faction. (@beccabunny)
+  * Bug fixes:
+    * Typo fixes. (@alextd, @Amazinite, @Arachi-lover, @Darcman99, @Gods-Righthand, @Mageking17, @MCOfficer, @oo13, @perey, @petervdmeer, @tehhowch, @Waffleship, @wjp, @Zitchas)
+    * The Quicksilver (Mark II) variant no longer has the exact same outfits as the base variant. (@Amazinite)
+    * The Shuttle and Behemoth no longer contact the image border. (@beccabunny) 
+    * Terraforming 7 will no longer offer on any planet. (@Bladewood)
+    * Explicitly stated the source of various missions so that they don't accidentally offer where they shouldn't. (@Amazinite)
+    * Pact Recon missions no longer off on pirate planets. (@Amazinite)
+    * The " Drak " government now has the correct swizzle. (@Zitchas)
+    * Added a tooltip for "movement:" that was missing. (@oo13)
+    * Fighters and drones that are able to jump will now correctly follow the player if left behind by the player. (@tehhowch)
+    * The "scram drive" attribute now displays with the same scale as the "jump speed" attribute. (@Amazinite)
+    * The Marauder Firebird's shield/hull ratio now match the Firebird's. (@Arachi-Lover)
+    * Fixed a missing null check. (@NomadicVolcano)
+    * Fixed a potential divide by zero error in the shipyard/outfitter. (@tehhowch)
+    * Missions will now properly indicate on the map if they have been completed. (@NomadicVolcano)
+  * Game content:
+    * New content:
+      * Various new Remnant missions, ships, and outfits. (@Zitchas, @Amazinite, images from @beccabunny)
+      * Various new spaceport missions in human space. (@petervdmeer)
+      * Added new hails to alien governments where they were missing some under certain conditions. (@Amazinite)
+      * Added new Pug and Heliarch hand to hand weapons. (@Brick63)
+      * Added new, larger Coalition shield and hull repair modules used only by the Heliarchs. (@Arachi-Lover, @beccabunny)
```

Comment:
To be more in line with the rest, you could use "Arachi-Lover, images by @beccabunny"